data/styles/clear/include/priorities_4_overlays.prio.txt

```diff
@@ -1239,6 +1242,15 @@ amenity-parking-park_and_ride                       # icon z16- (also has captio
 amenity-parking-underground                         # icon z16- (also has caption(optional) z18-)
 === 300
 
+entrance-main                                       # icon z18- (also has caption(optional) z18-)
+=== 280
```

Comment:
It makes the main entrance higher than building's name...
Let's put it just under buildings.

Reply:
fixed 👍